ci/test-pre.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+source ci/_
+
+annotate() {
+  ${BUILDKITE:-false} && {
+    buildkite-agent annotate "$@"
+  }
+}
+
+ci/affects-files.sh \
+  .rs$ \
+  Cargo.lock$ \
+  Cargo.toml$ \
+  ci/test-stable-perf.sh \
+  ci/test-stable.sh \
+|| {
+  annotate --style info \
```

Comment:
`test-stable-perf.sh` we want to skip, but `test-stable.sh` should always run even on a README.md change to catch cases where we edit some random data file that has a hidden dependency